config/environments/production.rb

```diff
@@ -56,8 +56,6 @@
   # Do not fallback to assets pipeline if a precompiled asset is missed.
   config.assets.compile = false
 
-  config.assets.prefix = '/dev-assets'
```

Comment:
Should definetely stay there or else the deployed version would not have the ProximaNova font